### DISCUSSION TITLE
can't figure out if I'm starting a manual install the correct way and if so how to set args

### DISCUSSION THREAD

**Discussion by deathtome1998** [Comfy-Org/ComfyUI#3591] [Q&A]
so I set up a manual install of comfyui, going through the git clone and pip install steps, for a windows PC with an NVIDIA rtx 4090. now my issue is, I'm not sure if using the command terminal to run python main.py in the main comfyui folder is the right way to start it for a manual install. if it is the right way, how do I set the args so it does the different vram levels if need be or any other ones I might need? also is there a way to make running the python main.py command easier if that's the correct way to start it? because opening the command terminal every time and putting in a command I'm never going to remember is not going to be fun.

**Comment by ltdrdata** [comment]
When you mention a manual install, do you mean you are not using the official package distributed with python_embeded?

**Reply by deathtome1998** [reply]
not sure what you mean by official package. I didn't do the portable standalone build if that's what you mean. I followed the steps that were lower down that specifically say manual install (windows, linux) but since I'm on windows I obviously followed the windows section of it.

**Reply by ltdrdata** [reply]
> not sure what you mean by official package. I didn't do the portable standalone build if that's what you mean. I followed the steps that were lower down that specifically say manual install (windows, linux) but since I'm on windows I obviously followed the windows section of it.

If you haven't installed the standalone package, you must create a venv and install it there. Installing any Python program on the system Python can lead to disaster.

**Reply by deathtome1998** [reply]
well crap, normally that's in the install instructions so I didn't even think of that. I take it I should just reinstall python then and then redo my pip stuff in a venv? which I can't remember how to do, so a bit of instruction there would be nice. or is the standalone the same as the manual in terms of functionality and updatedness? because if it is I might just reinstall python and grab the standalone one.

**Reply by ltdrdata** [reply]
The functionality and update capability of the standalone package are the same as with manual installation. I just use a venv because it is easier to manage.

**Reply by deathtome1998** [reply]
yeah, it might be, but I just decided to go ahead and reinstall python and grab the standalone package.

**Comment by tusharbhutt** [comment]
Here, this was much better at helping me out than the instructions on this GitHub.

https://medium.com/@yushantripleseven/installing-comfyui-linux-windows-b59a57af61b6
